motion_tracker/utils/crop_vis.py

```diff
@@ -1,7 +1,7 @@
 import cv2
 from motion_tracker.utils.image_cropping import Coords, crop_and_resize
 
-def show_img(window_name, img, msec_to_show_for=1500):
+def show_img(window_name, img, msec_to_show_for=4500):
     '''Display an image on the screen for fixed length of milliseconds'''
 
```

Comment:
Nothing to see here. Just changed for my own viz, and will `checkout` the original file before merging.